xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/futures/BinanceFuturesStreamingMarketDataService.java

```diff
@@ -0,0 +1,162 @@
+package info.bitrich.xchangestream.binance.futures;
+
+import com.google.common.util.concurrent.RateLimiter;
+import info.bitrich.xchangestream.binance.BinanceStreamingMarketDataService;
+import info.bitrich.xchangestream.binance.BinanceStreamingService;
+import info.bitrich.xchangestream.binance.BinanceSubscriptionType;
+import info.bitrich.xchangestream.binance.dto.BinanceWebsocketTransaction;
+import info.bitrich.xchangestream.binance.dto.DepthBinanceWebSocketTransaction;
+import io.reactivex.rxjava3.core.Flowable;
+import org.knowm.xchange.binance.BinanceErrorAdapter;
+import org.knowm.xchange.binance.dto.BinanceException;
+import org.knowm.xchange.binance.dto.marketdata.BinanceOrderbook;
+import org.knowm.xchange.binance.service.BinanceMarketDataService;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.marketdata.OrderBook;
+import org.knowm.xchange.exceptions.RateLimitExceededException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class BinanceFuturesStreamingMarketDataService extends BinanceStreamingMarketDataService {
```

Comment:
Seems like the entire class is duplicated because of the OrderbookSubscription.initiated field that is missing at super.
Can't we simply add this field to super to? Or find another workaround to prevent duplicating the class?